### ISSUE TITLE
Config For Pine Trees

### ISSUE THREAD

**Issue by KenyMylankca** [BlackJar72/ClimaticBiomePlacement#12]
can you add config for disabling pine trees please? some other mods already add pine&palm trees so i want to disable it. thanks!

**Comment by BlackJar72** [comment]
I suppose I could, though it would leave the subtropical forest as nothing but oaks.  Or, I could add an option to rename them to "cedars"...

I suppose an option wouldn't hurt.

EDIT: This could be harder than you think.  Taking out the pines would make at least one biome (the pinewoods) empty and strangely named.  Taking out pine biomes would mean anyone who selects that would need to edit those files too (and I guarantee someone wouldn't and would show up complaining about the crash).

**Comment by KenyMylankca** [comment]
hmmms... then make it different than other mods add :D

**Comment by BlackJar72** [comment]
Just which mod -- the tree and its blocks already look very different from [BoP pines](https://biomesoplenty.fandom.com/wiki/Pine_Tree) and different from any other Minecraft pine I know of (they should, I made my own assets).  They also live in completely different places, representing hot weather pines such as the loblolly, longleaf, sand pine, and slash pine -- other mods, especially BoP have mostly cold weather pines living in cool biomes.

I googled several word combinations involving "Minecraft" "pine" and "mod(ed)" and found nothing really similar.

...and why should it by **_MY_** trees that go and not their's -- I mean BoP is pretty big, but it's already established their pines are nothing like mine.
